components/style/properties/declaration_block.rs

```diff
@@ -673,10 +673,6 @@ impl ToCss for PropertyDeclarationBlock {
                         }
                     } else {
                         for (longhand, importance) in self.declaration_importance_iter() {
-                            if already_serialized.contains(longhand.id()) {
```

Comment:
Do we need to remove the same check in the system font loop?

(I don't remember why we have the special path for the system font in the first place...)

Reply:
It is redundant, yes.